test/test-utils/index.ts

```diff
@@ -18,7 +18,7 @@ import { Cli } from './cli'
 Object.assign(tinyrainbow.default, tinyrainbow.getDefaultColors())
 
 interface VitestRunnerCLIOptions {
-  std?: 'inherit'
+  std?: 'inherit' | 'tty'
```

Comment:
I added `std: inherit` just for debugging, maybe it should be renamed?

Reply:
Any suggestions what to call it?